### DISCUSSION TITLE
I need only 3 thing to change firefox to FLOORP

### DISCUSSION THREAD

**Discussion by cannnAvar** [Floorp-Projects/Floorp#1295] [Feature Request]
Hi I don't want to use firefox anymore because of mozila so i start to search. I download Floorp installer and I see 3 thing.
1. Installer only install system wide. Pleasee can we get user installation. It's better than portable download.
2. Export User Data from orijinal mozilla firefox. I just copy paste and profile folder and it work but why I can't export 1 click.
3. Many thing can't separate from mozilla. If i connect firefox account why i use floorp instead firefox

thank you for time for read my words.
Please notify me I realy want to use FLoorp

**Comment by ahmed-abdelrazek** [comment]
your number 3 question anwsers your number 2 problem

**Reply by cannnAvar** [reply]
Right i wasn't make question it self. Sory. But you get my point. Right?

**Reply by ahmed-abdelrazek** [reply]
if what you mean is can you move your firefox stuff to floorp with your mozilla account and enable sync yes you can i even have it between my firefox mobile floorp on my linux pc and windows laptop
but if what you are asking why use floorp instead of firefox if they are the same even floorp uses mozilla account my anwser would be for me at least
floorp offer many features out of the box which are not found in firefox by default like profiles managment - PWAs - old non button like tabs (themes) - workspaces and others in nice one easy install away package instead of having to configure any of that myself every time i install firefox

**Comment by MaxHasBeenUsed** [comment]
> Export User Data from orijinal mozilla firefox. I just copy paste and profile folder and it work but why I can't export 1 click.

Yeah I feel you about this. Floorp could build a slightly better profile manager, for simpler profile switch and profile migrate (possibly resolve conflicts). The profile manager Mozilla made can get work done but it's pretty basic. Although... I doubt many people would really use profile manager that often.

> Many thing can't separate from mozilla. If i connect firefox account why i use floorp instead firefox

Floorp is not a "de-Mozilla-lized" version of Firefox, it's a custom browser based on Firefox. Why I choose it? Because it can offer better performance and privacy than Firefox in certain circumstances. And why I didn't choose any de-Googled or de-Mozilla-lized browser? Because I definitely want a sync feature for a everyday browser. 

At least you can use Mozilla's sync feature in some Firefox based browsers, but you can never find Goolge's sync feature in any browser other than Chrome. And for small dev teams, sync required server cost and bandwidth cost is just too much to handle, so you can hardly find any custom browsers shipped with their own home-made sync feature.
